src/Compilers/Core/Portable/PEWriter/Members.cs

```diff
@@ -609,7 +609,7 @@ IPlatformInvokeInformation PlatformInvokeData
         /// <summary>
         /// Custom attributes associated with the method's return value.
         /// </summary>
-        IEnumerable<ICustomAttribute> ReturnValueAttributes { get; }
+        IEnumerable<ICustomAttribute> GetReturnValueAttributes(CommonPEModuleBuilder moduleBuilder);
```

Comment:
Consider passing an `EmitContext` argument instead, for consistency with other `Cci` interface methods.